### PR TITLE
Set checkout time and return time based upon default setting at create time

### PR DESCRIPTION
**Pull request by cjcolvar**
Related to #4740

### CODE REVIEW

File: app/models/checkout.rb [hunk 1]
```diff
@@ -3,11 +3,26 @@ class Checkout < ApplicationRecord
...
   scope :returned_for_user, ->(user_id) { where(user_id: user_id).where("return_time < now()") }
 
   def media_object
     MediaObject.find(media_object_id)
   end
+
+  private
+
+  def set_checkout_return_times
+    self.checkout_time = DateTime.now
```
Comment (masaball) [inline app/models/checkout.rb:19]: Should this be `DateTime.current`?

**Review by masaball — APPROVED** [review]
Looks good!

File: app/models/checkout.rb (hunk 1 above)
Comment (cjcolvar) [reply app/models/checkout.rb:19]: :100: Good catch!

**Review by masaball — APPROVED** [review]
Looks good.

**Pull request merged**